libs/template/templates/experimental-jobs-as-code/template/{{.project_name}}/src/dlt_pipeline.ipynb.tmpl

```diff
@@ -14,7 +14,7 @@
    "source": [
     "# DLT pipeline\n",
     "\n",
-    "This Delta Live Tables (DLT) definition is executed using a pipeline defined in resources/{{.project_name}}.pipeline.yml."
+    "This Lakeflow Declarative Pipeline (LDP) definition is executed using a pipeline defined in resources/{{.project_name}}.pipeline.yml."
```

Comment:
Please remove this acronym, we don't generally use it

Reply:
https://github.com/databricks/cli/pull/3487/files